sp/src/game/server/ai_speech_new.cpp

```diff
@@ -921,7 +925,7 @@ bool CAI_Expresser::SpeakDispatchResponse( AIConcept_t concept, AI_Response *res
 			textParams.g1 = 255;
 			textParams.b1 = 255;
 
-			if (ai_speech_print_mode.GetBool() && GetOuter()->GetGameTextSpeechParams( textParams ))
+			if (ai_speech_print_mode.GetBool() && GetOuter()->DispatchGetGameTextSpeechParams( textParams ))
```

Comment:
Do you have to know what this change does? I'm curious

Reply:
`GetGameTextSpeechParams` is used to get a NPC's speech params, and by default it can be overridden by VScript to have custom parameters. However, `GetGameTextSpeechParams` is also a virtual function which can be overridden by derived classes, and this overrides the VScript hook. This was a problem when I tried to use VScript to override the text params of Bad Cop or NPCs. To fix this without breaking existing overrides, I made the functions that display the text call a new `DispatchGetGameTextSpeechParams` function which checks for the VScript hook before using the overridable `GetGameTextSpeechParams`.

You can see more about this change in `baseflex.cpp`/`.h`, where the function is implemented.